units/src/fee.rs

```diff
@@ -134,6 +134,7 @@ impl FeeRate {
     /// This is equivalent to converting `vb` to [`Weight`] using [`Weight::from_vb`] and then calling
     /// `Self::fee_wu(weight)`.
     #[must_use]
+    #[deprecated(since = "TBD", note = "convert to a weight_unit and use `to_fee()` instead")]
```

Comment:
`Weight`, `weight_unit` doesn't exist.

Reply:
replaced with the message: `use Weight::from_vb and then to_fee() instead `